### ISSUE TITLE
[Feature Request] Replace abbrevations with icons

### ISSUE THREAD

**Issue by Madis0** [machiav3lli/oandbackupx#124]
**Is your feature request related to a problem? Please describe.**
Abbrevations "aren't translatable or don't make sense in the respective syntax of a language". #123 

**Describe the primer solution**
Remove abbrevations and use Material icons instead. The icons can use the same fill colors as the abbrevation badges use now. I am not sure how big they should be though... Feel free to use outline variants if you consider them better.

- USR - https://material.io/resources/icons/?icon=person
- SYS - https://material.io/resources/icons/?icon=settings_applications
- SPC - https://material.io/resources/icons/?icon=info
- APK - https://material.io/resources/icons/?icon=insert_drive_file
- DAT - https://material.io/resources/icons/?icon=storage
- A&D - no need, just use both icons side by side
- UPD - https://material.io/resources/icons/?icon=get_app

**Describe alternative solutions**
Colored dots or backgrounds of some kind.

**Screenshots**
N/A

**System Information(please complete the following information):**
N/A

-------------------------------
**Additional Notes**
* If you're on the latest Test version(from the TG-Channel or Matrix-Room) please write it there directly.
* If you're on the latest github version. Please read all the open Issues if one does match yours, leave a comment there if there's some extra information you can provide.
* If you're not on the latest github version(happens sometimes with F-Droid version), please read all of the open issues and also some of the latest closed issues.

Thanks for your suggestion

**Comment by machiav3lli** [comment]
Thanks for the suggestion, I'll test it while working on the next version and see how it works and the feedback it gets.

**Comment by hg42** [comment]
I would suggest
SYS https://material.io/resources/icons/?icon=settings
clearer for this size

UPD https://material.io/resources/icons/?icon=update or
UPD https://material.io/resources/icons/?icon=new_releases
should be better visible, more emphazised

SPLITAPK https://material.io/resources/icons/?icon=call_split
SPLITAPK https://material.io/resources/icons/?icon=qr_code is more similar to APK but the dots may be too  small
or
APK https://material.io/resources/icons/?icon=stop or another single square
SPLITAPK https://material.io/resources/icons/?icon=widgets

I also agree with Kannan on Telegram, who misses the color coding.

My colors would be like that:
basically green is "good" and red is "dangerous" etc., so may be:

SYS purple (royal, spritual)
USR orange (skin)
SPC yellow (star)
APK lightblue (sky) or mint?
DAT green (safe)
UPD lightred (unsafe, warning)

**Comment by Madis0** [comment]
> My colors would be like that:
basically green is "good" and red is "dangerous" etc., so may be:

Yet you make update dangerous and data safe? That makes no sense.
If anything, system and special should be "dangerous" and user, update "safe".

**Comment by hg42** [comment]
apk or data stands for "is already in the backup" which is safe.
"updated" really means "changed", which is apk or data is changed in the system and still not in the backup, which I call unsafe.
If everything is in the backup, the list will have all lightblue and green on the left.
If something is not in the backup you see a red "updated" icon.

**Comment by hg42** [comment]
system is purple for that matter and special yellow, both far from green.
But the icon for "still to be backuped" should be more red.

The mixed usage of the color for state and type is a bit of a problem.
It would be nicer to see "all is green" so the backup is complete.
And if something is red, you have to look at it.

The backup state could additionally be indicated by the background color of the whole entry.
